src/content/post/20161122_drupal_application_bottlenecks_avoid_with_profiling.md

```diff
@@ -0,0 +1,152 @@
++++
+date = "2016-11-22T11:02:30Z"
+draft = false
+title = "Avoid bottlenecks on a Drupal web application and how identify it with profiling"
+description = "How identify bottleneck of your web application and avoid that can appear in the future"
+tags        = [ "profiling", "drupal", "performance", "web application"  ]
+topics      = [ "Profiling", "Drupal", "Performance" ]
+author = "Vinz"
++++
+
+### TL;DR
+
+This article is focused on the possible *bottlenecks of a web application*, how you can identify it to improve the performance of your code.
+
+We talk about profiling with *Blackfire*, PHP web app focused on *Drupal*. LAMP is the reference stack.
+
+<!-- more -->
+
+Application's performance is one of characteristics what separates a job from a good job, and demonstrates quality work.
```

Comment:
`is one of characteristics what separates a job from a good job` -> `is one of the feature that tells a *done* job from a *good* job, and shows a focus on quality.